exporter/elasticsearchexporter/internal/objmodel/objmodel_test.go

```diff
@@ -40,37 +40,45 @@ func TestObjectModel_CreateMap(t *testing.T) {
 		},
 		"from map": {
 			build: func() Document {
-				return DocumentFromAttributes(pcommon.NewMapFromRaw(map[string]interface{}{
+				m := pcommon.NewMap()
+				m.FromRaw(map[string]interface{}{
 					"i":   42,
 					"str": "test",
-				}))
+				})
+				return DocumentFromAttributes(m)
 			},
 			want: Document{[]field{{"i", IntValue(42)}, {"str", StringValue("test")}}},
 		},
 		"ignores nil values": {
 			build: func() Document {
-				return DocumentFromAttributes(pcommon.NewMapFromRaw(map[string]interface{}{
+				m := pcommon.NewMap()
+				m.FromRaw(map[string]interface{}{
 					"null": nil,
 					"str":  "test",
```

Comment:
Separate PR, maybe use `UpsertX` for the values?

Reply:
Makes sense, updated here since it's still related